src/components/donation-popup.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Link } from 'gatsby';
+import logo from "../images/ARLogo-full-color.png";
+import cover from "../images/smile-3.jpg";
+import { CSSTransition } from 'react-transition-group';
+
+const DonationPopup = () => {
+  const [showPopup, setShowPopup] = useState(false);
+  const nodeRef = useRef(null);
+
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      setShowPopup(true);
+    }, 500);
+
+    return () => clearTimeout(timeoutId);
+  }, []);
+
+  // When the popup is open, prevent scroll on the page
+  useEffect(() => {
+    if (showPopup) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = 'auto';
+    }
+  }, [showPopup]);
+
+  return (
+    <CSSTransition in={showPopup} timeout={500} classNames="fade" unmountOnExit>
+      <div style={{ zIndex: "100", position: "relative" }} ref={nodeRef}>
+        <div style={{ display: "flex", alignItems: "center", justifyContent: "center", backgroundColor: "#002C6D", minHeight: "100vh", minWidth: "100vw", position: "fixed" }}>
```

Comment:
is there a way to still show the regular website content in the background? or would that look bad?

Reply:
We can, though it looks a bit strange because you have two photos of you:

<img width="1455" alt="image" src="https://github.com/ashwinforga/ashwinforgeorgia.com/assets/3037552/4f6b1016-3cd2-4d84-962e-ec2558b15b12">

That said, it comes with the benefit of making it clear that it's a popup, which might be helpful to less tech literate users for whom that may not be clear with the opaque background.

Reply:
Let's do the popup one with partial opacity! Thanks.